.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,45 @@
+<!-- Thank you for contributing to our WordPress open source project! -->
+
+## Description
+<!-- Please provide a clear and concise description of your changes -->
+<!-- What problem does this PR solve? What functionality does it add/improve? -->
+
+## Related Issue
+<!-- Please link any related issues here. Use the syntax: "Fixes #123" or "Related to #456" -->
+
+## Dependant PRs
+<!-- List any dependent PR's that are awaiting review. Use the following syntax: "#1234" -->
```

Comment:
I think these can be one section? `## Related Issues and PRs` And prompt them to describe the relationship

Reply:
I believe keeping "Related Issue" and "Dependent PRs" separate might be beneficial as they serve different purposes and improve clarity. Related Issues connect to the problem being solved, while Dependent PRs track technical merge dependencies. I'll enhance both sections to prompt for relationship context, including BEFORE/WITH/AFTER dependency types with helpful examples.